.cargo/config.toml

```diff
@@ -2,12 +2,9 @@
 target = "riscv64gc-unknown-none-elf"
 
 [target.riscv64gc-unknown-none-elf]
-rustflags = ["-C", "link-arg=-Tlinker.ld"]
+rustflags = [ "-C", "code-model=medium"]
 runner = """
-  qemu-system-riscv64 \
-    -machine virt \
-    -bios none \
-    -smp 1 \
-    -nographic \
-    -kernel
+  qemu-system-riscv64 -machine virt
+    -display none -serial stdio -smp 2
```

Comment:
Can we just stick to a single CPU core for now?

Reply:
And we should add [this](https://docs.amd.com/r/en-US/ug1304-versal-acap-ssdg/Symmetric-Multiprocessing) as a reference for someone who wants to understand - `what's SMP`.